dev-client/src/store/reducers.ts

```diff
@@ -39,6 +40,7 @@ const sliceReducers = {
   site: siteReducer,
   project: projectReducer,
   soilId: soilIdReducer,
+  sync: syncReducer,
```

Comment:
Don't forget to add the sync slice to the load-from-storage routine in `persistence`.